components/brave_new_tab_ui/components/default/braveToday/options/customize.tsx

```diff
@@ -27,6 +27,7 @@ const CustomizeButton = styled('button')<Props>`
   border: none;
   font-weight: 800;
   cursor: pointer;
+  pointer-events: ${p => p.show ? 'auto' : 'none'};
```

Comment:
@petemill  I confirmed this change fixed the issue. WDYT?

Reply:
Thanks I didn't notice this. It's good for now, I'll rebase on the latest refactor.